lms/resources/_js_config.py

```diff
@@ -357,6 +370,14 @@ def _blackboard_files_enabled(self):
         """
         return self._application_instance().settings.get("blackboard", "files_enabled")
 
+    def _onedrive_files_enabled(self):
+        """
+        Return True if the OneDrive Files API is enabled for this request.
+
+        :raise ConsumerKeyError: if request.lti_user.oauth_consumer_key isn't in the DB
+        """
+        return self._application_instance().settings.get("onedrive", "files_enabled")
```

Comment:
Ok, so we're enabling the OneDrive feature per application instance. That makes sense because we can enable it only for our test application instances during development, and then only for certain customers. Eventually we might want to enable it by default